src/main/java/hinlok/Hinlok.java

```diff
@@ -0,0 +1,42 @@
+package hinlok;
+
+import hinlok.command.Command;
+import hinlok.parser.Parser;
+import hinlok.tasks.TaskList;
+import hinlok.file.TaskFile;
+import hinlok.ui.Ui;
+
+public class Hinlok {
+    private Ui ui;
+    private TaskList taskList;
+    private TaskFile taskFile;
+
+    public Hinlok(String savedFilePath) {
+        ui = new Ui();
+        taskFile = new TaskFile(savedFilePath);
+        taskList = new TaskList(taskFile.loadTaskFromFile());
+    }
+
+    public static void main(String[] args) {
+        Hinlok hinlok = new Hinlok("./data/hinlok.txt");
+        hinlok.chat();
+    }
+
```

Comment:
Seems to me that you are missing the Storage class specified in the instructions. Is there any reason you decided against implementing it?